lib/perl/Genome/Model/Tools/Picard/MarkDuplicates.pm

```diff
@@ -7,46 +7,44 @@ use Genome;
 
 my $DEFAULT_ASSUME_SORTED = 1;
 my $DEFAULT_REMOVE_DUPLICATES = 0;
-my $DEFAULT_MAX_RECORDS_IN_RAM = 500000;
 
 class Genome::Model::Tools::Picard::MarkDuplicates {
-    is  => 'Genome::Model::Tools::Picard',
+    is  => 'Genome::Model::Tools::Picard::Base',
     has_input => [
         input_file => {
             is  => 'String',
             doc => 'The SAM/BAM files to merge.  File type is determined by suffix.',
+            picard_param_name => 'INPUT',
         },
         output_file => {
             is  => 'String',
             doc => 'The resulting merged SAM/BAM file.  File type is determined by suffix.',
+            picard_param_name => 'OUTPUT',
         },
         metrics_file => {
             is  => 'String',
             doc => 'File to write duplication metrics to.',
+            picard_param_name => 'METRICS_FILE',
         },
         assume_sorted => {
-            is  => 'Integer',
-            valid_values => [1, 0],
+            is  => 'Boolean',
             doc => 'Assume the input data is sorted.  default_value='. $DEFAULT_ASSUME_SORTED,
             default_value => $DEFAULT_ASSUME_SORTED,
             is_optional => 1,
+            picard_param_name => 'ASSUME_SORTED',
         },
         remove_duplicates => {
-            is => 'Integer',
-            valid_values => [1, 0],
+            is => 'Boolean',
             doc => 'Merge the seqeunce dictionaries. default_value='. $DEFAULT_REMOVE_DUPLICATES,
```

Comment:
It's not something you changed, but these doc lines don't need to include the defaults since the help text will already show them.   Additionally this doc line doesn't match the parameter it is attached to!